lib/inspec/plugin/v2/loader.rb

```diff
@@ -170,6 +170,23 @@ def self.find_gem_directory(gem_name, version = nil)
       selected_gemspec && selected_gemspec.full_gem_path
     end
 
+    def self.find_gemspec_directory(gem_name, version = nil)
+      matching_gem_versions = list_managed_gems.filter { |g| g.name == gem_name }.sort(&:version)
```

Comment:
okay so this clearly doesn't capture all installed gems. `list_managed_gems` only lists gems for which the gemspec is available in `~/.inspec/gems/specifications`. But the resource packs itself can be installed in the environment and need not be in the plugin dir.

Reply:
Oh is it? I did not see that part of logic where the resource packs are installed in the environment. Is that via manual intervention?